web-app/js/portal/filter/DateFilter.js

```diff
@@ -123,17 +122,14 @@ Portal.filter.DateFilter = Ext.extend(Portal.filter.Filter, {
     _getDateString: function(newDate) {
 
         if (newDate) {
-            newDate.setHours(this.timezoneCorrection);
```

Comment:
### To test what is happening

Using the workflow that was given for bug #800

1) Select collection IMOS - ANMN National Reference Stations - Maria and North Stradbroke Islands - Near real-time meteorology and sea surface temperature
2) On Step 2, select 2012/11/16 to 2012/11/17 in temporal extent

### What Happens - On https://github.com/aodn/aodn-portal/pull/1980 pull request

* Step 3 shows:

```
Temporal Extent: 2012/Nov/16-00:00-UTC to 2012/Nov/17-00:00-UTC
```

* Downloaded data as CSV has timestamps from '2012-11-15T13:06:49' to '2012-11-16T12:06:50'


=========================================
On Production

* Step 3 shows:

```
Temporal Extent: 2012/Nov/16-10:00-UTC to 2012/Nov/17-10:00-UTC
```

*This is incorrect. The user would expect that they are picking from the start of 2012/Nov/16 to the start of 2012/Nov/17 all in UTC*


* Downloaded data as CSV has timestamps from  '2012-11-15T23:06:20' to '2012-11-16T22:06:49'